CHANGELOG.md

```diff
@@ -1,44 +1,50 @@
 # Changelog
 
```

Comment:
@DominikGuzei 
This PR is meant to be a hot fix that will be included in the next Daedalus `release/0.12.0`. I just went ahead and assigned the next version as `0.7.2` as it is targeting the master branch. Feel free to change this if needed!